### DISCUSSION TITLE
Option to hide decimals for input_number

### DISCUSSION THREAD

**Discussion by rabbit52068** [home-assistant/frontend#11285] [Other feature requests]
## The request
Lovelace Entity Card:
Option to show the input_number's state without any decimal places.
Currently the Entity Card always shows 1 decimal place when the nominated entity is input_number.

## The alternatives

Create a template sensor to read the state from the target input_number entity to show in the Entity Card.

**Comment by spacegaier** [comment]
The frontend is showing whatever the entity is providing as value/state. An `input_number` by default has a decimal. So the frontend is not adding anything here, but your entity just has that value.
